app/controllers/account/registrations_controller.rb

```diff
@@ -35,8 +35,10 @@ def destroy
       not_found
     end
 
+    # rubocop:disable Metrics/AbcSize
     def phone_verification
-      phone = current_user[:mobile_phone] = params.dig(:user, :mobile_phone)
+      phone = current_user[:mobile_phone] = TelephoneNumber.parse(params.dig(:user, :mobile_phone),
```

Comment:
`TelephoneNumber` kitaplığını ayrıntılı incelemedim.  Burada hata yönetimimiz var mı veya gerek var mı?  Örneğin `.parse` metodunun döndüğü nesne `.valid?` metoduna [cevap veriyor](https://github.com/mobi/telephone_number/blob/master/lib/telephone_number/class_methods.rb#L15).

Reply:
`valid?` olup olmadığı `User` modelindeki `mobile_phone` ile ilgili validasyon tarafından sağlanıyor